src/building-blocks/FormField.tsx

```diff
@@ -29,6 +29,7 @@ class FormField extends Component<IFormFieldProps> {
       ;
 
     return {
+      disabled: fieldConfig.disabled,
```

Comment:
Why is this necessary? won't this be in `editProps`?

Reply:
Na, it's a core attribute:
![image](https://user-images.githubusercontent.com/796717/76240648-f7cf3100-6209-11ea-872c-1d306e52b381.png)